azurefunctions-extensions-bindings-blob/tests/test_blobclient.py

```diff
@@ -142,13 +142,13 @@ def test_invalid_input_populated(self):
 
         with self.assertRaises(ValueError) as e:
             datum: Datum = Datum(value=sample_mbd, type="model_binding_data")
-            result: BlobClient = BlobClientConverter.decode(
+            _: BlobClient = BlobClientConverter.decode(
                 data=datum, trigger_metadata=None, pytype=BlobClient
             )
```

Comment:
You can directly call BlobClientConverter instead of assigning it to _

```suggestion
            BlobClientConverter.decode(
                data=datum, trigger_metadata=None, pytype=BlobClient
            )
```

Reply:
Will look at changing in another PR